README.md

```diff
@@ -32,34 +33,28 @@ There are 2 types of helper classes in this app:
 - [AppHelpers](lib/govuk_publishing_components/app_helpers). Are exposed to the applications using this gem. They should be documented using RDoc.
 - [Component Presenters](lib/govuk_publishing_components/presenters). Anything in these classes is only for use within the components. They should be marked `@private`.
 
-## GOV.UK Frontend
+## Relationship with GOV.UK Design System
 
-This gem is the way GOV.UK consumes [GOV.UK Frontend](https://github.com/alphagov/govuk-frontend), part of the GOV.UK Design System.
+GOV.UK Publishing Components also makes [GOV.UK Design System](https://design-system.service.gov.uk/) styles and components available to GOV.UK frontend applications. This gem consumes [GOV.UK Frontend](https://github.com/alphagov/govuk-frontend) via [NPM](https://www.npmjs.com/).
 
-We use NPM to pull in the GOV.UK Frontend styles and Javascript, but we don't commit
-the `node_modules` directory. To install GOV.UK Frontend on your machine, run:
-
-```
-npm install
-```
-
-You'll need to have installed Node and NPM for this.
+## Browser and assistive technology support
+GOV.UK Publishing Components shares the same standards in terms of browser and assistive technology support with [GOV.UK Frontend](https://github.com/alphagov/govuk-frontend#browser-and-assistive-technology-support).
```

Comment:
Is this true? We support IE8 but your project could decide to follow the service manual?

Reply:
This update is followed by a discussion between frontend developers in GOV.UK that happened yesterday. The reasoning behind this is that, as far as I know, the decision for the current browser matrix that GOV.UK Frontend uses was based on data from GOV.UK.

Reply:
Happy for this matter to be discussed more widely if we think otherwise – only that I currently can't make any of the frontend meetings 😞.

Reply:
> the decision for the current browser matrix that GOV.UK Frontend uses was based on data from GOV.UK.

GOV.UK Frontend's matrix is based on supporting the lowest denominator across *all* services. Services themselves should follow the service manual as a basis and adjust according to the specific audience.

Reply:
We already state this in the [GOV.UK developer docs](https://docs.publishing.service.gov.uk/manual/browser-support.html) it seems so this change shouldn't be controversial.